src/jabs/ui/video_list_widget.py

```diff
@@ -54,6 +55,12 @@ def _selection_changed(self, current, previous):
         """ signal main window that use changed selected video """
         if current:
```

Comment:
Does this need to have an `else` branch for when current doesn't evaluate to True? e.g. Should `self._selected_video` be set to `None`?